units/crons.go

```diff
@@ -414,6 +414,22 @@ func PopulateOldestImageRemovalJobs() amboy.QueueOperation {
 	}
 }
 
+func PopulateCommitQueueJobs() amboy.QueueOperation {
+	return func(queue amboy.Queue) error {
+		catcher := grip.NewBasicCatcher()
+		ts := util.RoundPartOfHour(1).Format(tsFormat)
```

Comment:
if you actually want this to run every minute, the current way that the scheduling works, you'll *probably* run it slightly less in some situations. if you think that'll cause latency problems, let's talk about ways to clear that up.

Reply:
seems okay to me. I doubt the queue will get too long even if it's running every couple of minutes.